### ISSUE TITLE
.gitignore file required

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/ddablib#11]
Lots of temporary files are showing up as modified and cluttering up the list of unstaged files.

**Comment by delphidabbler** [comment]
Fixed in `develop` at commit d5ee356

Issue will be closed once the commit is merged into `main`.

**Comment by delphidabbler** [comment]
There are changes afoot that mean that this new `.gitignore` file will have limited use.
It is planned to revert this commit.

**Comment by delphidabbler** [comment]
Commit reverted at commit 39751ce

Closing this issue now
